main.tex

```diff
@@ -215,198 +222,254 @@ \subsection*{Functional variation}
+Codon \texttt{402} also occurs within a trans-membrane segment (\texttt{I.S6}), and the \texttt{V402L} substitution has been independently associated with pyrethroid resistance in bedbugs \cite{Yoon2008}.
 %
-The remaining 3 variants are within trans-membrane domains, and so may enhance resistance by altering or interacting with the insecticide binding sites on the VGSC \cite{Dong2014}.
+Other substitutions at this locus have also been associated with resistance, \texttt{V402A/G} in the moth crop pests \emph{Helicoverpa zea} \cite{Hopkins2010} and \texttt{V402M} in \emph{Heliothis virescens}, the latter of which has been shown experimentally to confer resistance in \textit{Xenopus} oocytes \cite{Park1997, Lee2013}.
+%
+@@TODO Aedes combination.
+%
+Because of the limited geographical distribution of these alleles, we hypothesize that the \texttt{I1527T+V402L} combination represents a pyrethroid resistance allele that arose in West African \textit{An. coluzzii} populations.
 %
-Because of the tight linkage between these 13 SNPs and the \texttt{L995F} allele, we classify all as putative \texttt{L995F} enhancers, although experimental work is required to confirm a resistance phenotype.
+However, the \texttt{L995F} allele is at higher frequency (85\%) in our Burkina Faso \textit{An. coluzzii} population, and is known to be increasing in frequency \cite{Toe2014}, therefore \texttt{L995F} may provide a stronger resistance phenotype and is replacing \texttt{I1527T+V402L}.
 %%
 
-%% Paragraph 5 - Other high frequency variants we think are probably not resistance variants and why.
-%
-The remaining 3 variants (\texttt{M490I}, \texttt{A1125V}, \texttt{V1254I}) do not occur in combination with any known resistance allele, and do not appear to be associated with haplotypes under selection \cite{Ag1000gConsortium2017}
+
+%% Other novel variants.
 %
-A possible exception is the \texttt{M490I} allele found at 18\% frequency in the Kenyan population, although the fact that this population has experienced a recent population crash makes it difficult to test for evidence of selection at this locus.
+The remaining 4 novel alleles (two separate nucleotide substitutions causing \texttt{M490I}; \texttt{A1125V}; \texttt{V1254I}) do not occur in combination with any known resistance allele.
 %
-All 3 variants occur in internal linker domains, and so do not fit the model of a resistance driver, again experimental work is required to rule out a resistance phenotype.
+All are private to a single population, and (to our knowledge) none have previously been found in other species.
 %
+All occur within internal linker domains of the protein and are therefore unlikely to directly cause resistance, discussed further below.
 %%
 
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection*{Haplotype structure}
+\subsection*{Genetic backgrounds carrying resistance alleles}
```

Comment:
I've had a go at the subsection on genetic backgrounds / haplotype clusters. The main thing I thought to do here is incorporate the results from the dxy scan and clustering by shared haplotype length, because they speak directly to our classification of different genetic backgrounds. However, in order to avoid this getting too long-winded and interrupting the flow, I have dealt with these relatively briefly, and was going to put the corresponding figures into supplementary material.